less/lists.less

```diff
@@ -403,54 +397,23 @@
 
 #feed .list-item {
   margin-left: 0;
-  margin-top: 10px;
-
-  .list-item-info {
-    min-height: 60px;
-  }
   &.waypoints .waypoint-type {
     flex-basis: 25%;
     @media @phone {
       flex-basis: 0;
     }
   }
   &.outings {
-    .list-item-info {
-      padding-left: 30%;
-    }
+    border-color: @C2C-orange;
     .activities {
       margin-bottom: 0;
     }
-    .icon-user {
-      margin-bottom: 5px;
-      width: 100%;
-      font-size: 30px;
-      text-align: center;
-    }
-    .outing-author, .date {
-      left: 10px;
-      width: 25%;
-      position: absolute;
-      justify-content: center;
-      .flex();
-    }
+
     .outing-author {
-      top: 33%;
-      flex-flow: wrap;
-      align-content: flex-start;
-      justify-content: center;
-    }
-    .author-name {
-      line-height: normal;
-      word-break: break-all;
-      word-break: break-word; // non-standard for webkit
-      text-align: center;
+      display: none;
```

Comment:
Could this `display: none` be simply replaced by removing the content? Or is this something shown in the feed but not in lists (or vice versa)?